wrist_adduction_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: wrist_adduction
    joint_speed: 5
    motor:
        id: 32
        init: 2048
        min: 0
        max: 4095